jenkins/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM ubuntu
```

Comment:
I would suggest using a standard `jenkins/jenkins` image and installing the plugin on the top of it.

https://github.com/oleg-nenashev/demo-jenkins-config-as-code/blob/master/Dockerfile does plugin installation using `plugins.txt`, but you can just copy the HPI to a plugin directory